priv/templates/skf.gen.makefile/Makefile

```diff
@@ -16,69 +17,85 @@
 
 DEFAULT_GOAL: help
 
-<%= if cloud == "heroku" do %>
-DEVELOPMENT?=https://git.heroku.com/<%= app %>_backend-develop.git
-STAGING?=https://git.heroku.com/<%= app %>_backend-stage.git
-PRODUCTION?=https://git.heroku.com/<%= app %>_backend-prod.git
-<% end %>
-
+# TODO read database name from Mix.Config
 PGDATABASE?=<%= app %>_backend_dev
 
 export LOCAL_USER_ID ?= $(shell id -u $$USER)
 
-# ============= #
-# POOR MANS CLI #
-# ============= #
+# -------------------
+# --- DEFINITIONS ---
+# -------------------
 
 require-%:
 	@ if [ "${${*}}" = "" ]; then \
 		echo "ERROR: Environment variable not set: \"$*\""; \
 		exit 1; \
 	fi
 
+# -----------------
+# --- MIX TASKS ---
+# -----------------
+
 dev:
 	@MIX_ENV=dev mix ecto.setup && iex -S mix phx.server 
 
-heroku-console:
-	@MIX_ENV=dev heroku run "iex -S mix" --remote develop
-	
-heroku-logs:
-	@MIX_ENV=dev heroku logs --tail --remote develop
-
 test:
 	@MIX_ENV=test mix test
 
 compile:
 	@mix do deps.get, compile
 
-capture-backup: require-REMOTE
-	heroku pg:backups:capture --remote ${REMOTE}
-
-latest.dump: require-REMOTE
-	heroku pg:backups:download --remote ${REMOTE}
-
-import-latest-db: require-PGDATABASE latest.dump
-	dropdb --if-exists ${PGDATABASE} && \
-	createdb ${PGDATABASE} && \
-	pg_restore --verbose --clean --no-acl --no-owner --dbname=${PGDATABASE} latest.dump
-
 reset-db:
-	mix do ecto.reset
+	@mix do ecto.reset
 
 seed-db:
-	mix seed
+	@mix seed
 
 reseed-db: reset-db seed-db
 
+<%= if cloud && cloud == "heroku" do %>
+# --------------
+# --- HEROKU ---
+#  -------------
+
+DEVELOPMENT?=https://git.heroku.com/<%= app %>_backend-develop.git
+STAGING?=https://git.heroku.com/<%= app %>_backend-stage.git
+PRODUCTION?=https://git.heroku.com/<%= app %>_backend-prod.git
+
+## Set Heroku remotes
 set-remotes: require-DEVELOPMENT require-STAGING require-PRODUCTION
```

Comment:
These tasks should be included in (or excluded from) PHONY depending on the user configuration. If I understand correctly, this is also what @ademirovic mentioned in the review comment.